maintainers/release-process.md

```diff
@@ -159,6 +148,30 @@ release:
 
 ## Creating a point release
 
+* Checkout.
+
+  ```console
+  $ git checkout XX.YY-maintenance
```

Comment:
```suggestion
  $ git checkout X.YY-maintenance
```

Reply:
I guess I was looking forward to Nix 10.0.